utf8.h

```diff
@@ -150,7 +150,7 @@ utf8_nonnull utf8_pure utf8_weak void *utf8valid(const void *str);
 // Sets out_codepoint to the next utf8 codepoint in str, and returns the address
 // of the utf8 codepoint after the current one in str.
 utf8_nonnull utf8_weak void *utf8codepoint(const void *utf8_restrict str,
-                                           long *utf8_restrict out_codepoint);
+                                           int *utf8_restrict out_codepoint);
```

Comment:
Your thoughts on this change? I see no reason to return a variable of a type that could potentially be 8 bytes while we only need 4!